### DISCUSSION TITLE
I Nedd 1.12.2

### DISCUSSION THREAD

**Discussion by aa710411** [Pokecube-Development/Pokecube-Issues-and-Wiki#1147] [General]
I Nedd 1.12.2
I encountered a bottleneck while expanding this version,

**Comment by Thutmose** [comment]
1.12.2 support ended many many years ago, the codebase was a mess however, feel free to try to get it working:

https://github.com/Thutmose/Pokecube
https://github.com/Thutmose/Pokecube-Mobs
https://github.com/Thutmose/Thut/tree/1.12.x/ThutCore

Those should have most of the needed stuff there.
